tests/test_dia_object.py

```diff
@@ -42,6 +42,9 @@ def create_test_dia_sources(n_sources=5):
     -------
     A lsst.afw.SourceCatalog
     """
+    if schema is None:
+        schema = schema
```

Comment:
What's going on here? This line makes no sense.

Reply:
Oops, I wanted this to be make_minimal_dia_source_schema(). It is now fixed.